src/content/en/2020/css.md

```diff
@@ -115,22 +126,40 @@ Only a few of the top classes were presentational, with most of those being eith
 
 Despite IDs being discouraged these days in some circles due to their much higher specificity, most websites still use them, albeit sparingly. Fewer than half of pages used more than one ID in any of their selectors (had a max specificity of (1,x,y) or less) and nearly all had a median specificity that did not include IDs (0,x,y).
 
+{# TODO Authors—Can you explain this (1,x,y)/(0,x,y) syntax more? #}
+
 But what are these IDs used for? It turns out that the most popular IDs are structural: `#content`, `#footer`, `#header`, `#main`, despite [corresponding HTML elements](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Document_and_website_structure#HTML_layout_elements_in_more_detail).
 
+{# TODO Authors—This sentence just ends. Do you mean "despite corresponding HTML elements existing that could be used as selectors while also improving the semnatic markup"? #}
+
 {{ figure_markup(
   image="popular-ids.png",
   caption="The most popular IDs by the percent of pages.",
-  description="Bar chart showing the top 10 most popular IDs for desktop and mobile pages. The most popular ID is content at 14% of pages. The footer and header IDs have slightly smaller adoption. The logo, main, respond, comments, fancybox-loading, wrapper, and submit IDs have between 5 and 10% adoption on pages. The only notable difference between desktop and mobile is the comments ID used on about 8% of mobile pages but only 5% of desktop pages.",
+  description="Bar chart showing the top 10 most popular IDs for desktop and mobile pages. The most popular ID is `content` at 14% of pages. The `footer` and `header` IDs have slightly smaller adoption. The `logo`, `main`, `respond`, `comments`, `fancybox-loading`, `wrapper`, and `submit` IDs have between 5 and 10% adoption on pages. The only notable difference between desktop and mobile is the `comments` ID used on about 8% of mobile pages but only 5% of desktop pages.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRpe_HsNGpekn6YZV9k6QGmcZPxalqnDrL7DrDY-7X65RZEf_-aGfWuEvhk-yWV83ctIceE1bppCLpj/pubchart?oid=141873739&format=interactive",
   sheets_gid="734822190",
   sql_file="top_selector_ids.sql"
 ) }}
 
 IDs can also be used to intentionally reduce or increase specificity. The specificity hack of writing an ID selector as an attribute selector (`[id="foo"]` instead of `#foo`) was surprisingly rare, with only 0.3% of pages using it at least once. Another ID-related specificity hack, using a negation + descendant selector like `:not(#nonexistent) .foo` instead of `.foo` to increase specificity, was also very rare, appearing in only 0.1% of pages.
 
-### !important
+{# TODO Authors: Oh clever! I'd never heard of this! Presumably that first one is to reduce specificty? Should we make that clearer? Or link to an article/articles about either or both of these hacks for uninitiated like me? #}
+
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
yes - "instead of `#foo`) to reduce specificity was surprisingly rare"

I suggest [specificity hack](https://csswizardry.com/2014/07/hacks-for-dealing-with-specificity/)

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/1846/commits/c90c8fd4362f57f70b4e331a83bde12d73e4a209